README.md

```diff
@@ -1,148 +1,133 @@
 
 ## Setup
 
-1. [Download your Twitter archive.](https://help.twitter.com/en/managing-your-account/how-to-download-your-x-archive) 
+1. [Download your Twitter archive.](https://help.twitter.com/en/managing-your-account/how-to-download-your-x-archive)
+
+`More (3 dot button) > Settings and Privacy > Your Account > Download an archive of your data`
+
+Note: It takes around 1-2 days for twitter to prepare your archive data.
 
-    `More (3 dot button) > Settings and Privacy > Your Account > Download an archive of your data`
+Clone the repository:
 
-   It takes around 1-2 days.
+```bash
+git clone https://github.com/sankalp1999/twitter-circle.git --depth 1
+```
 
-    Clone the repository:
-    ```
-    git clone https://github.com/sankalp1999/twitter-circle.git
-    ```
+1. Copy your Twitter archive into the project folder and rename the archive/zip file to `twitter-archive`. You  have to ensure the archive is on the root folder of the project.
 
-2.  Copy your Twitter archive into the project folder and rename the archive/zip file to `twitter-archive`
-You just have to ensure the archive is on the root folder of the project.
+1. Install Node.js (for Linux and MacOS)
 
+- Visit the official Node.js website: <https://nodejs.org>
+- Download the appropriate version for your operating system
+- Follow the installation instructions provided on the website
 
-3. Install Node.js (for Linux and macOS):
-   - Visit the official Node.js website: https://nodejs.org
-   - Download the appropriate version for your operating system
-   - Follow the installation instructions provided on the website
+1. Install project dependencies
 
+```bash
+npm install
+```
 
-4. Install project dependencies:
-   ```
-   npm install
-   ```
-   This command will install all the necessary packages listed in the `package.json` file.
+This command will install all the necessary packages listed in the `package.json` file.
 
-5. Set up the project (Linux and macOS):
-   ```
-   chmod +x setup.sh
-   ./setup.sh
-   ```
-   This will make the `setup.sh` script executable and run it to set up the project.
+1. Set up the project (Linux and macOS)
 
+```bash
+./setup.sh
+```
 
-6. Start the application:
-   ```
-   npm start
-   ```
-   This command will start the Twitter Circle application.
+1. Start the application
 
+```bash
+npm start
+```
 
+This command will start the Twitter Circle application.
 
 You may see some timeouts/errors for the profile picture script. You can still proceed
 or if too many pictures are missing, please re-run the script.
 
-Customer support: early access users contact me [here](https://twitter.com/dejavucoder)
-
-P.S
+Customer support: early access users contact me [https://twitter.com/dejavucoder](https://twitter.com/dejavucoder)
 
-If you have already cloned, just git pull once to fetch latest updates.
-`git pull`
+NOTE: If you have already cloned, run `git pull` to fetch latest updates.
```

Comment:
not sure if I should keep this line in general or not. I have added it to setup.sh

Reply:
yet to push that change.